retrieval/scrape.go

```diff
@@ -285,13 +285,15 @@ type targetScraper struct {
 }
 
 const acceptHeader = `application/vnd.google.protobuf;proto=io.prometheus.client.MetricFamily;encoding=delimited;q=0.7,text/plain;version=0.0.4;q=0.3,*/*;q=0.1`
+const userAgentHeader = `Prometheus-scraper/1.0`
```

Comment:
I thought about that too, but examples like Go-http-client/1.1 or Googlebot/2.1 made me think there might be another versioning logic for scrapers. Updated nevertheless.